wiki/Introduction-to-Creating-a-Job.md

```diff
@@ -0,0 +1,1818 @@
+create a Job Template from those shell scripts so that you can run the Job on your distributed
+computing infrastructure. The Job Template describes the shape of the Job, its runtime environment,
+and the processes that will run.
+
+### 2.2.1. Embed the scripts into a Job Template
+
+Starting with a skeleton of the minimal required properties of a Job Template:
+
+```yaml
+specificationVersion: jobtemplate-2023-09
+name: 
+steps:
+  - name: 
+    script:
+      actions:
+        onRun:
+          command: 
+```
+
+Fill in the names for the Job that you're going to create, and the first Step that will run the Blender
+rendering script:
+
+```yaml
+specificationVersion: jobtemplate-2023-09
+name: DemoJob
+steps:
+  - name: BlenderRender
+    script:
+      actions:
+        onRun:
+          command: 
+```
+
+The way to think about a Step in Open Job Description is that it is defining a command to run and the collection of inputs
+to run the command with. This is a [map operation](https://en.wikipedia.org/wiki/Map_(higher-order_function)), if
+you are familiar with that concept. You're not going to define any inputs for the command yet, so you'll be using the default
+of having a single Task with empty input. The command is defined in the `onRun` Action in the skeleton above; this is
+the command that is run when running a Task within a [Session](How-Jobs-Are-Run#sessions). An Action in the 2023-09 
+revision of the specification [is defined as](2023-09-Template-Schemas#5-action):
+
+```yaml
+command: <CommandString> # @fmtstring[host]
+args: [ <ArgString>, ... ] # @optional @fmtstring[host]
+timeout: <posinteger> # @optional
+cancelation: <CancelationMethod> # @optional
+```
+
+Notice that both the `command` and elements of the `args` array are annotated with `@fmtstring[host]` indicating that
+they are [Format Strings](How-Jobs-Are-Constructed#format-strings) that are evaluated on the host that the command will
+run on. We're going to make use of this property to create our Job.
+
+You've already expressed your Job as a series of two shell scripts to run in sequence, so next you'll make use of the
+[Embedded File](2023-09-Template-Schemas#6-embeddedfile) concept to make those scripts available to the Job. An embedded file
+lets you define the contents of a text file directly in a Job Template, then that text file will be created in the 
+[Session's temporary Working Directory](How-Jobs-Are-Run#sessions) when the Task is run.
+
+The result is the start of your Job Template, which we save to file called `job.template.yaml`:
+
+```yaml
+specificationVersion: jobtemplate-2023-09
+name: DemoJob
+steps:
+  - name: BlenderRender
+    script:
+      actions:
+        onRun:
+          # Note: Task.File.Render refers to the file location on disk where the
+          #  contents of the embedded file named "Render" are materialized.
+          command: "{{Task.File.Render}}"
+          args:
```

Comment:
Be good to comment here that this *needs to be a list* as it's one area we've been people butt their heads against (including me)